docs/design_proposals/ko-builder.md

```diff
@@ -285,87 +290,176 @@ maps directly to this value.
-    file?
-
-    Suggest yes, to make Skaffold a compelling choice for Go developers.
-    __Not Yet Resolved__
+    Suggest embedding as a Go module.
 
-3.  Should Skaffold use base image settings from
+2.  Should Skaffold use base image settings from
     [`.ko.yaml`](https://github.com/google/ko#configuration) if the ko builder
     definition in `skaffold.yaml` doesn't specify a base image?
 
-    Suggest yes to simplify adoption of Skaffold for existing ko users.
-    __Not Yet Resolved__
+    __Resolved:__ Yes, to simplify adoption of Skaffold for existing ko users.
+
+3.  If a config value is set both as an environment variable, and as a config
+    value, which takes precedence? E.g., `ko.sourceDateEpoch` vs
+    `SOURCE_DATE_EPOCH`.
+
+    __Resolved:__ Follow existing Skaffold patterns.
 
 4.  Should the ko builder have a config option for
     [`SOURCE_DATE_EPOCH`](https://reproducible-builds.org/specs/source-date-epoch/),
     or should users specify the value via an environment variable?
 
-    __Not Yet Resolved__
+    __Resolved__: Specify via the reproducible builds spec environment variable
+    `SOURCE_DATE_EPOCH`, see
+    <https://github.com/google/ko#why-are-my-images-all-created-in-1970> and
+    <https://reproducible-builds.org/docs/source-date-epoch/>.
 
-5.  If a config value is set both as an environment variable, and as a config
-    value, which takes precedence? E.g., `ko.sourceDateEpoch` vs
-    `SOURCE_DATE_EPOCH`.
 
-    Follow existing Skaffold pattern - is there one? __Not Yet Resolved__
 
-6.  Add a Google Cloud Build (`gcb`) support for the ko builder?
+### Open questions
+
+1.  Should we default dependency paths to `{"go.mod", "**.go"}` instead of
+    `{"."}`.?
+
+    The former is a useful default for many (most?) Go apps, and it's used
+    in the `custom` example. The latter is the default for some other builders.
+
+    __Not Yet Resolved__
+
+2.  Add a Google Cloud Build (`gcb`) support for the ko builder?
+
     Other builders that support `gcb` have default public builder images.
     The image `gcr.io/tekton-releases/ko-ci` is public, but do we want to
     rely on it? Once ko is embedded in Skaffold, we could use
     `gcr.io/k8s-skaffold/skaffold` as a default image.`
 
     __Not Yet Resolved__
 
-7.  File sync support: Should we limit this to
+3.  File sync support: Should we limit this to
     [ko static assets](https://github.com/google/ko#static-assets) only?
 
     This is the only way to include additional files in a container image
     built by ko.
 
     __Not Yet Resolved__
 
-8.  Should we default dependency paths to `{"go.mod", "**.go"}` instead of
-    `{"."}`.?
-
-    The former is a useful default for many (most?) Go apps, and it's used
-    in the `custom` example. The latter is the default for some other builders.
+4.  Should the ko builder be the default for `skaffold init`, instead of
+    buildpacks, for Go apps, when there's no Dockerfile and no Bazel workspace
+    file?
```

Comment:
I think this would be a good options. The way `init` treats buildpacks currently is pretty much like a catch all if we can't find a more specific builder to use for the project. So ko would be a good option to use first in this case